web/lib/utils.ts

```diff
@@ -91,3 +91,8 @@ export const checkUserPermissions = (
   }
   return validRoles.includes(membership?.role);
 };
+
+export const getNullifierName = (nullifier: string | undefined | null) => {
+  if (!nullifier) return null;
+  return `Anonymous User | ...${nullifier.slice(-5)}`;
```

Comment:
why do we need to say anonymous user. Can we just do a 0x{nullifier} or something

Reply:
Updated